server/reports/commission/commission-report.js

```diff
@@ -1,20 +1,22 @@
-// module "commission-report"
+//module "commission-report"
 import json2csvExports from "json2csv";
+import {printAuthorizationKey, CommissionReport} from "../../commision.js";
 import MindbodyAccess from "../../api-manager.js";
 
-/* A handy request URLS for testing:
-http://localhost:8080/reports/commission?format=csv&startdate=111&enddate=2222
+/*a handy request URLS for testing:
+http://localhost:8080/reports/commission?format=csv&startdate=2020-03-01T00:00:00&enddate=2020-03-031T00:00:00
 */
 export function handleCommissionRptRequest(request, response) {
 
-  //console.log("handleExpressRequest:");
+  //console.log("handleCommissionRptRequest:");
   //console.log("request: " + request.baseUrl + " - " + JSON.stringify(request.query));
 
-  // Validate input parameters
+  //validate input parameters
   //query string looks like: ?format=[json|csv]&startdate=[whatever]&enddate=[whatever]
   let format = request.query.format;
   if (!format) {
-    format = "json";
+    response.send("Missing \"startdate\" parameter.");
```

Comment:
"Format" should be required. I just chose a dumb default before, but now it's required.